LLMonFHIR/FHIRSummary/FHIRResourceSummary.swift

```diff
@@ -67,7 +79,26 @@ final class FHIRResourceSummary: Sendable {
     func summarize(resource: FHIRResource, forceReload: Bool = false) async throws -> Summary {
         let resource = try resource.copy()
         try? resource.stringifyAttachements()
-        return try await resourceProcessor.process(resource: resource, forceReload: forceReload)
+
+        var retryCount = 0
+        var summary: Summary?
+
+        repeat {
+            summary = try await resourceProcessor.process(resource: resource, forceReload: forceReload || retryCount > 0)
+            retryCount += 1
+
+            guard summary == nil else {
+                break
+            }
+
+            try? await Task.sleep(for: .seconds(0.5))
```

Comment:
Wondering if we want to reduce the timeout here.

Reply:
I lowered it to 0.1 seconds.